### ISSUE TITLE
Incompatible with uvloop==0.9.1

### ISSUE THREAD

**Issue by rilut** [squeaky-pl/japronto#115]
```
MacBook-Pro:test rilut$ pip3 install  uvloop==0.9.1
Collecting uvloop==0.9.1
  Using cached uvloop-0.9.1-cp36-cp36m-macosx_10_11_x86_64.whl
Installing collected packages: uvloop
  Found existing installation: uvloop 0.8.1
    Uninstalling uvloop-0.8.1:
      Successfully uninstalled uvloop-0.8.1
Successfully installed uvloop-0.9.1

MacBook-Pro:test rilut$ python3 japro.py
Accepting connections on http://0.0.0.0:8080
Exception in callback UVTransport._call_connection_made
handle: <Handle UVTransport._call_connection_made>
Traceback (most recent call last):
  File "uvloop/cbhandles.pyx", line 52, in uvloop.loop.Handle._run
  File "uvloop/handles/tcp.pyx", line 149, in uvloop.loop.TCPTransport._call_connection_made
  File "uvloop/handles/basetransport.pyx", line 156, in uvloop.loop.UVBaseTransport._call_connection_made
  File "uvloop/handles/basetransport.pyx", line 153, in uvloop.loop.UVBaseTransport._call_connection_made
SystemError: NULL result without error in PyObject_Call

MacBook-Pro:test rilut$ pip3 install  uvloop==0.8.1
Collecting uvloop==0.8.1
  Downloading uvloop-0.8.1-cp36-cp36m-macosx_10_11_x86_64.whl (1.2MB)
    100% |████████████████████████████████| 1.2MB 748kB/s
Installing collected packages: uvloop
  Found existing installation: uvloop 0.9.1
    Uninstalling uvloop-0.9.1:
      Successfully uninstalled uvloop-0.9.1
Successfully installed uvloop-0.8.1

MacBook-Pro:test rilut$ python3 japro.py
Accepting connections on http://0.0.0.0:8080
```

**Comment by rilut** [comment]
uvloop==0.9.1 is installed while I was doing `pip3 install -U sanic`, because now sanic can use `uvloop` too

**Comment by martmists** [comment]
See #99 and #107 

setup.py has been adjusted to specifically only install uvloop<0.9.0, so if you use uvloop above this version you're indeed going to have issues.

**Comment by 0x161e-swei** [comment]
Is there any plan to upgrade to uvloop 0.9.1?

**Comment by ifplusor** [comment]
I fixed it in #133
